src/herbivoresAndCarnivores.test.js

```diff
@@ -34,8 +34,7 @@ describe('Animal', () => {
 
       expect(carn)
         .toHaveProperty('health', 100);
-    }
-  );
+    });
 
   it(`Herbivore should have 'hide' method which inverts 'hidden' property`,
```

Comment:
The 'hide' method test description suggests it toggles the 'hidden' property, but according to the checklist, it should only set this property to true.